engine/src/main/java/org/terasology/logic/characters/ServerCharacterPredictionSystem.java

```diff
@@ -121,12 +127,20 @@ public void onPlayerInput(CharacterMoveInputEvent input, EntityRef entity) {
         }
         CircularBuffer<CharacterStateEvent> stateBuffer = characterStates.get(entity);
         CharacterStateEvent lastState = stateBuffer.getLast();
-
-        float delta = input.getDelta() + lastState.getTime() - (time.getGameTimeInMs() + MAX_INPUT_OVERFLOW);
+        float delta = input.getDelta() + lastState.getTime() - (time.getGameTimeInMs() + MAX_INPUT_OVERFLOW + 100);
```

Comment:
What's with the `100` here? Magic number (should be a constant) and doesn't that effectively double the max overflow? Is that okay?